CHANGELOG.md

```diff
@@ -34,8 +36,25 @@ and includes an additional section for migration notes.
 
 ### Migration Notes
 
+- Create a new bucket `PREFIX-orca-reports` in the same account and region as your primary orca bucket.
+  - Give the bucket a [lifecycle configuration](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-set-lifecycle-configuration-intro.html) with an expiration period of 30 days.
+  - Follow instructions in https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-s3-bucket/ to set up permission policy.
+  - Modify the permissions for your primary Orca bucket.
+    - Under the `Cross Account Access` policy, add `s3:GetInventoryConfiguration` and `s3:PutInventoryConfiguration` to Actions.
```

Comment:
how does this work? once the above actions are added to the `Cross Account Access` policy, how does it actually update the policy in AWS?

Reply:
Theoretically, they could just add these to the policy in AWS, unless their account has more restrictions around that, but in sandbox. I was just able to edit the policy in the AWS GUI.

Reply:
sounds good.